src/main/java/assets/game.js

```diff
@@ -113,11 +113,14 @@ function cellClick() {
     let col = String.fromCharCode(this.cellIndex + 65);
     let useSonar = document.getElementById("use_sonar").checked;
     if (isSetup) {
+        if(shipType === "SUB" && document.getElementById("place_submarine_underwater").checked){// Flag as an underwater sub
+            shipType = "SUBB";
+        }
```

Comment:
Makes sense.  Glad you created a separation between above and bellow water submarine.